internal/resources/policy/type/security/scope_schema.go

```diff
@@ -0,0 +1,154 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package security
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	policyclustermodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/cluster"
+	policyclustergroupmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/clustergroup"
+	policyorganizationmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/organization"
+	scoperesource "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/scope"
+)
+
+var scopeSchema = &schema.Schema{
+	Type:        schema.TypeList,
+	Description: "Scope for the policy",
```

Comment:
can you add the accepted scope values in the description so it comes out  well in the document

Reply:
Sure

Reply:
Done